### ISSUE TITLE
when we add a product attribute as a Media Image type. I created the attribute successfully through the backend.But it is not displaying in the Add Product page in the back end.

### ISSUE THREAD

**Issue by deepakk-webkul** [magento/magento2#26021]
Preconditions (*)
Magento CE 2.3.3 
PHP 7.2.24

Steps to reproduce (*)
1.  Create a Media Image type attribute
2.  Add this attribute in a related product attribute set.
3.  Observe that in product this attribute is not added.

Expected result (*)
1. Custom Media Image type attribute should be displayed on the product details page.

Actual result (*)
1. Custom Media Image type attribute does not display on the Product Details Page.

**Comment by ajijshekh123** [comment]
Give me @magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @ajijshekh123. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @ajijshekh123, here is your [Magento instance](https://i-26021-2-4-develop.instances.magento-community.engineering).
Admin access: [https://i-26021-2-4-develop.instances.magento-community.engineering/admin](https://i-26021-2-4-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Echo 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-29756` were created

**Issue Available**: @engcom-Echo, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by dharmendra-wagento** [comment]
@magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @dharmendra-wagento. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @dharmendra-wagento, here is your [Magento instance](https://i-26021-2-4-develop.instances.magento-community.engineering).
Admin access: [https://i-26021-2-4-develop.instances.magento-community.engineering/admin_76a7](https://i-26021-2-4-develop.instances.magento-community.engineering/admin_76a7)
Login: `d1d27ab6` Password: `38dc154c1e0e`
Instance will be terminated in up to 3 hours.

**Comment by dharmendra-wagento** [comment]
Hi @magento-engcom-team ,

Thank you for providing the instance.

I have followed the above steps and tried to re-produce it in the given instance but it's not re-produced. The custom media image is showing properly in front side in product media gallery area.

Herewith I have attached the screen-shot for admin and front-end for the same.

![image-gallery-front](https://user-images.githubusercontent.com/20059461/73604296-5fafad80-45b4-11ea-8625-f5b049da7ab3.png)
![image-gallery-admin](https://user-images.githubusercontent.com/20059461/73604297-5fafad80-45b4-11ea-97b6-795533aa8b7c.png)
